apolloschurchapp/src/tabs/home/Features/ActionListCardFeature/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+
+import { styled, ActionListCard, H3, H6 } from '@apollosproject/ui-kit';
+
+const Title = styled(
+  ({ theme }) => ({
+    color: theme.colors.text.tertiary,
+  }),
+  'ActionListCardFeature.Title'
+)(H6);
+
+const Subtitle = styled({}, 'VerticalCardListFeature.Subtitle')(H3);
+
+const ActionListCardFeature = memo(
```

Comment:
I might name this component `ActionListFeature` to match the feature `__typename`, rather than matching the component it renders. We just happen to use the `ActionListCard` for the action list feature, in the same way we happen to use the `FeedView` for the vertical card list feature.

Reply:
Makes sense to me. Fixed in 17982a6